docs/source/science/properties/mobility.rst

```diff
@@ -0,0 +1,103 @@
+.. _science_properties_mobility:
+
+Charge Carrier Mobility
```

Comment:
We will provide electron and hole mobility as two different "Properties", computed by the same calculator, right? So if the user initiates a computation for "Electron Mobility" choosing this property from a dropdown in the frontend, he should find the label "Electron Mobility" somewhere in the title to make the connection. Suggestion: "Electron and Hole Mobility" or "Charge carrier mobility: electron mobility and hole mobiliy". The more similar the strings, the more easy it is to connect between frontend and documentation.

Reply:
changed to Electron and Hole Mobility